libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.spec.ts

```diff
@@ -1587,6 +1587,33 @@ describe('Text editor', () => {
       );
     }));
 
+    it('should render help inline popover', () => {
+      testComponent.helpPopoverContent = 'popover content';
+      testComponent.labelText = 'label text';
+
+      fixture.detectChanges();
+
+      expect(
+        fixture.nativeElement.querySelectorAll(
+          'sky-help-inline:not(.sky-control-help)',
+        ).length,
+      ).toBe(1);
+    });
+
+    it('should not render help inline popover if title is provided without content', () => {
```

Comment:
the logic that is being tested with this unit test is handled by help inline and not text editor so should we test this here? If we change something in help inline to break this, this unit test also failing would make narrowing down what broke messy

Reply:
I copied these from a few of the other implementations:
https://github.com/blackbaud/skyux/pull/2247/files#diff-5aa22d73575718f884930e39e9a9bbfecb660ca10ef9950b29615bebf6d9c000
https://github.com/blackbaud/skyux/pull/2272/files#diff-b97e172eacec04c7a16bb3261be8cdfc85a3f6a67d5c62c9779112b9756a0d7e
https://github.com/blackbaud/skyux/pull/2263/files#diff-9aa58baa3790535321d5d9a985d46788c56ac8f7cf0167f51e368ad44384ea46
https://github.com/blackbaud/skyux/pull/2262/files#diff-6b9228c4ee40fc8e73b1710797f54fa2db9d9871e515e9df9bdea679ceadd663
Also, this is testing logic that exists in the text editor's template, so it should be covered (e.g. `*ngIf="helpPopoverContent"`).

Reply:
oop i missed that, yeah we need this unit test in this case. disregard!